docs/music.rst

```diff
@@ -95,18 +94,14 @@ Functions
     their defaults before the music (whatever it may be) is played.
 
     An optional argument to specify the output pin can be used to override the
-    default of ``microbit.pin0``. With micro:bit **V2** you can use
-    ``microbit.pin_speaker`` or a tuple of two pins, for example
-    ``pin=(pin_speaker, pin0)`` which would output sound on the built-in
-    speaker and pin 0.
+    default of ``microbit.pin0``.
```

Comment:
This applies to all the modules: we should also indicate that if we don't want sound output to any pin we can pass `pin=None`

Reply:
Updated in https://github.com/bbcmicrobit/micropython/pull/695/commits/05c8b169faa12a8b27fa760a3f374dd47bc999a1